src/libs/model/functions.ts

```diff
@@ -0,0 +1,187 @@
+import {
+  IElement,
+  IPackage,
+  IClassifier,
+  IGeneralization,
+  IContainer,
+} from '@types';
+import { OntoUMLType } from '@constants/.';
+import memoizee from 'memoizee';
+
+export default {
+  IElement_functions: {
+    getRootPackage: getRootPackage,
```

Comment:
You can replace `getRootPackage: getRootPackage,` for only `getRootPackage,`. Do the same for the others